### PR TITLE
move interop data outside of static (fixes #3132)

### LINKED ISSUE #3132: Interop 2023: "Math Functions" could be labelled as "CSS Math Functions"?
At first, I thought that was about some methods in the ECMAScript `Math` object. People may understand that's CSS, but it could be better if being more precise. What do you think?

### PR DESCRIPTION
**Pull request by racheljay**
## Description
This PR moves the data that was previously in `components/static/interop-data.json` to a `.js` file outside of the static directory as outlined in the discussion for #3142.

## Changes
`components/static/interop-data.json` was previously removed and this PR adds that data to `components/interop-data.js`. It is now imported directly into `interop.js` and accessed by `fetchYearData()`.

### CODE REVIEW

File: webapp/components/interop-data.js [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+      'interop-2023-webcomponents': {
+        'description': 'Web Components',
+        'mdn': 'https://developer.mozilla.org/docs/Web/Web_Components',
+        'spec': 'https://www.w3.org/wiki/WebComponents/',
+        'tests': '/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-webcomponents',
+        'countsTowardScore': true
+      }
+    }
+  }
+};
```
Comment (DanielRyanSmith) [inline webapp/components/interop-data.js:540]: EOF new line here

**Review by DanielRyanSmith — COMMENTED** [review]
[Staging environment](https://move-interop-data-dot-wptdashboard-staging.uk.r.appspot.com) looks as expected. We should keep an eye on this during deployment to make sure this truly avoids the caching issue we encountered.
I'll let @KyleJu give final approvals since he discovered the root cause of the problem to begin with.

**Comment by KyleJu** [comment]
https://move-interop-data-dot-wptdashboard-staging.uk.r.appspot.com/interop-2023 works as expected

**Review by KyleJu — APPROVED** [review]

**Comment by KyleJu** [comment]
@racheljay Thank you for the PR!

**Comment by nt1m** [comment]
Should we delete the JSON file to avoid confusion?

**Pull request merged**